pages/index.vue

```diff
@@ -71,7 +71,22 @@
 <script lang="ts">
 import Vue from 'vue'
 
-export default Vue.extend({})
+export type DataType = {
+  todoList: []
+}
```

Comment:
怜央のコードで試して見たけど同じエラーは出てこなかったよ🤔
todoListの型エラーだけ出てたからこんな感じで書き換えたらエラー消えました

```suggestion
import { Todo } from '../store'

export type DataType = {
  todoList: Todo[]
}
```